package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ts-firebase-driver-testing",
-  "version": "0.28.8",
+  "version": "0.28.9",
```

Comment:
I stupidly published a tag with the wrong version the other day - you might need to delete it and republish https://github.com/freetrade-io/ts-firebase-driver-testing/releases/tag/v0.28.9 :sorry:!